Makefile

```diff
@@ -39,16 +39,27 @@ USER             := $(shell id -u -n)
 export PATH
 export GOBIN
 
+.PHONY: dev-setup
+dev-setup:
+	@./hack/dev-setup
+
 .PHONY: dev
 dev:
-	$(eval LD_FLAGS_RUN = "-w -X $(REPOSITORY)/pkg/version.Version="$(shell ./hack/get-next-version))
-	@KUBECONFIG=~/.kube/config GARDENER_KUBECONFIG=~/.kube/config WATCH_NAMESPACE=$(USER) go run -ldflags $(LD_FLAGS_RUN) cmd/gardener-controller-manager/main.go --config=dev/componentconfig-gardener-controller-manager.yaml
-
-.PHONY: dev-all
-dev-all:
 	$(eval LD_FLAGS_RUN = "-w -X $(REPOSITORY)/pkg/version.Version="$(shell ./hack/get-next-version))
 	@KUBECONFIG=~/.kube/config GARDENER_KUBECONFIG=~/.kube/config go run -ldflags $(LD_FLAGS_RUN) cmd/gardener-controller-manager/main.go --config=dev/componentconfig-gardener-controller-manager.yaml
 
+.PHONY: dev-apiserver
+dev-apiserver:
+	@go run cmd/gardener-apiserver/main.go \
+			--admission-control=ShootSeedFinder,ShootDNSHostedZone,ShootQuotaValidator \
```

Comment:
Can we add the `ShootSeedProtector` admission controller (similar to https://github.com/gardener/gardener/blob/master/charts/gardener/templates/deployment-apiserver.yaml#L38)?